src/main.rs

```diff
@@ -66,14 +66,18 @@ pub fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     // spin up a single-threaded tokio runtime to run the config loading task to completion
     // we don't want to wrap our whole program with a runtime since iced starts its own.
-    let config_load = {
+    let (config_load, window_load) = {
         let rt = runtime::Builder::new_current_thread()
             .enable_all()
             .build()?;
 
-        rt.block_on(Config::load())
-    };
+        rt.block_on(async { 
+            let config = Config::load();
+            let window = data::Window::load();
 
+            join!(config, window)
+        })
+    };
```

Comment:
We're already inside an `async` block here, let's just `.await` each one, no need for `join`